src/applications/mhv/secure-messaging/tests/e2e/fixtures/draftsResponse/multi-draft-response.json

```diff
@@ -20,7 +20,7 @@
         "threadId": 3103881,
         "folderId": -2,
         "messageBody": "draft #33",
-        "draftDate": "2023-12-22T23:11:42.000Z",
+        "draftDate": "2024-01-10T23:11:42.000Z",
```

Comment:
@oleksii-morgun-va The `secure-messaging-multiple-drafts-interface.cypress.spec.js` test started failing today when it couldn't find the text area to edit a message. After some digging I saw tha the input field is after a condition that checks for `cannotReply`. This will be true after 45 days based on this line in the message dispatch:

```
cannotReply: isOlderThan(lastSentDate, 45),
```

https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/mhv/secure-messaging/actions/messages.js#L102

I've updated your test data in this PR so I can get it to pass but this will probably fail again after another 45 days expire.